src/components/ListEntry/ListEntry.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { css } from 'glamor';
+import { Link } from 'react-router-dom';
+
+export default function ListEntry({ children, id, to, style, ...props }) {
```

Comment:
Why `...props`? It's not being used for anything.

Why not just get rid of id/to/style and pass them down as `{...props}` below instead?